code/debugconsole/console.cpp

```diff
@@ -488,6 +488,21 @@ void dc_putc(char c)
 	line_str->push_back(c);
 }
 
+bool handle_textInputEvent(const SDL_Event& event) {
+	if (event.text.text[0] != '\0' && event.text.text[0] != '\1') {
+		for (char c : event.text.text) {
+			if (c < 32)
```

Comment:
Hmmm... I'm assuming this check is to omit control/escape characters in the ASCII table, but I'm not entirely sure.

Reply:
Yup, all characters less than 32 are special: https://github.com/z64555/fs2open.github.com/pull/4/files#r546547542